Src/BootCamp.Chapter/Logging/LogToText.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter.Logging
+{
+    abstract class LogToText : ILog, IConnection
+    {
+
+        private string _Connection = @"C:\Users\Max\Source\Repos\CSharp-From-Zero-To-Hero\Src\BootCamp.Chapter\Logging\Log.txt";
+        public string GetConnection()
+        {
+            return _Connection;
+        }
+
+        public void LogOpenProgram()
+        {
+            LogNow($"Program Started. Time: {DateTime.Now}");
+        }
+        public void LogCloseProgram()
+        {
+            LogNow($"Program Stopped. Time: {DateTime.Now}");
+        }
+        public void LogCrash(Exception e)
+        {
+            LogNow($"Program Crashed Time: {DateTime.Now}\r\nReason: {e.Message}");
+        }
+        public void LogNow(string text)
+        {
+            using (var fileS = new FileStream(GetConnection(), FileMode.Create))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this code to make sure 'fileS' is disposed only once.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5150047)